src/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -131,16 +134,19 @@ public HttpRequestMessage()
 
         public HttpRequestMessage(HttpMethod method, Uri requestUri)
         {
-            if (NetEventSource.IsEnabled) NetEventSource.Enter(this, method, requestUri);
+            if (NetEventSource.IsEnabled)
+                NetEventSource.Enter(this, method, requestUri);
             InitializeValues(method, requestUri);
-            if (NetEventSource.IsEnabled) NetEventSource.Exit(this);
+            if (NetEventSource.IsEnabled)
+                NetEventSource.Exit(this);
```

Comment:
These should not be changed.  Our style throughout the project is for these to be one-liners, with the "if (NetEventSource.IsEnabled) " effectively just a prefix.

Reply:
This auto formatting was done by VS.